docs/running-tests/from-local-system.md

```diff
@@ -152,7 +152,7 @@ customising the test run:
     ./wpt run --help
 
 [A complete listing of the command-line arguments is available
-here](command-line-arguments.md).
+here](command-line-arguments.md#run).
```

Comment:
I checked the build output and this actually broke stuff:
```diff
-<p><a class="reference internal" href="command-line-arguments.html"><span class="doc">A complete listing of the command-line arguments is available
-here</span></a>.</p>
+<p><a class="reference external" href="command-line-arguments.md#run">A complete listing of the command-line arguments is available
+here</a>.</p>
```

Reply:
I found there are a few cases of broken links like this already that also need fixing...

Reply:
Sent https://github.com/web-platform-tests/wpt/pull/28831, will apply the same fix here.